docs/sources/helm-charts/mimir-distributed/run-production-environment-with-helm/_index.md

```diff
@@ -227,6 +227,11 @@ cluster, refer to
 
 For OpenShift-specific instructions see [Deploy on OpenShift](#deploy-on-openshift).
 
+The `mimir-distributed` Helm chart also deploys most of the containers
+with a read-only root filesystem (`readOnlyRootFilesystem: true`).
+The exceptions are the optional MinIO and Grafana Agent containers.
```

Comment:
it's still using the grafana agent operator. Do we call it alloy? I don't know

Reply:
Nah, they're separate binaries and Docker images